components/lark-pretty-print/src/lib.rs

```diff
@@ -27,28 +27,28 @@ impl PrettyPrint for Ty<Declaration> {
 }
 
 impl PrettyPrint for Ty<FullInferred> {
-    fn pretty_print(&self, db: &impl PrettyPrintDatabase) -> String {
+    fn pretty_print(&self, db: &(impl PrettyPrintDatabase + ?Sized)) -> String {
```

Comment:
this is the annoying thing -- by default, `impl Foo` requires `Sized`, so if we have a `dyn Blah`, we need to do `(impl Foo + ?Sized)`.